vue/components/ui/molecules/scrollable-items/scrollable-items.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+    <div class="scrollable-items">
+        <button-icon
+            class="button-icon-previous"
+            v-bind:icon="'chevron-left'"
+            v-bind:disabled="isButtonPreviousDisabled"
+            v-bind="_buttonIconProps"
+            v-if="showArrows"
+            ref="button-icon-previous"
+            v-on:click="onButtonIconPreviousClick"
+        />
+        <div
+            class="items-container"
+            v-bind:style="itemsContainerStyle"
+            ref="items-container"
+            v-on:wheel="onItemsContainerWheel"
+            v-on:mousedown="onItemsContainerMouseDown"
+        >
+            <div
+                class="item"
+                v-bind:class="itemClasses(item, index)"
+                v-for="(item, index) in items"
+                v-bind:key="item.value"
+                v-on:mouseup="onItemMouseUp($event, item)"
+            >
+                <slot v-bind:item="item">
+                    <slot v-bind:name="item.value" v-bind:item="item">
+                        {{ item.label || item.value }}
+                    </slot>
+                </slot>
+            </div>
+        </div>
+        <button-icon
+            class="button-icon-next"
+            v-bind:icon="'chevron-right'"
+            v-bind:disabled="isButtonNextDisabled"
+            v-bind="_buttonIconProps"
+            v-if="showArrows"
+            ref="button-icon-next"
+            v-on:click="onButtonIconNextClick"
+        />
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.scrollable-items {
+    align-items: center;
+    display: flex;
+    user-select: none;
+}
+
+.scrollable-items > .items-container,
+.scrollable-items > .items-container > .item {
+    cursor: pointer;
+    display: inline-block;
+}
+
+.scrollable-items > .items-container {
+    flex: 1;
+    -ms-overflow-style: none;
+    overflow-x: scroll;
+    scrollbar-width: none;
+    white-space: nowrap;
+}
+
+.scrollable-items > .items-container::-webkit-scrollbar {
+    display: none;
+}

```

Comment:
```
-ms-overflow-style: none;
overflow-x: scroll;
scrollbar-width: none;
```
and
```
.scrollable-items > .items-container::-webkit-scrollbar {
    display: none;
}
```

were added to have the default scrollbar behaviour in mobile, but without showing the scrollbar.